front/src/routes/integration/all/zigbee2mqtt/setup-page/SetupTab.jsx

```diff
@@ -0,0 +1,262 @@
+                          style={{ color: '#00f', backgroundColor: '#00f', borderColor: '#00f', height: 2, width: 40 }}
+                        />
+                      )}
+                      {props.zigbee2mqttRunning && props.zigbee2mqttConnected && (
+                        <i style={{ color: '#0f0', fontSize: '24px' }} class="fe fe-check" />
+                      )}
+                      {props.zigbee2mqttRunning && !props.zigbee2mqttConnected && (
+                        <i style={{ color: '#f00', fontSize: '24px' }} class="fe fe-x" />
+                      )}
+                      {props.zigbee2mqttRunning && (
+                        <hr
+                          style={{ color: '#00f', backgroundColor: '#00f', borderColor: '#00f', height: 2, width: 40 }}
+                        />
+                      )}
+                    </td>
+                    <td class="text-center">
+                      {props.zigbee2mqttExist && (
+                        <img src={logoZigbee2mqtt} alt={`Zigbee2mqtt`} title={`Zigbee2mqtt`} width="80" height="80" />
+                      )}
+                    </td>
+                  </tr>
+                  <tr>
+                    <td class="text-center">
+                      <div class="tag tag-success">
+                        <Text id={`systemSettings.containerState.running`} />
+                      </div>
+                    </td>
+                    <td class="text-center" />
+                    <td class="text-center">
+                      {props.mqttRunning && (
+                        <span class="tag tag-success">
+                          <Text id={`systemSettings.containerState.running`} />
+                        </span>
+                      )}
+                    </td>
+                    <td class="text-center" />
+                    <td class="text-center">
+                      {props.zigbee2mqttRunning && (
+                        <span class="tag tag-success">
+                          <Text id={`systemSettings.containerState.running`} />
+                        </span>
+                      )}
+                    </td>
+                  </tr>
+                </tbody>
+              </table>
+            </div>
+          </div>
+          <div class="card-header d-sm-none">
+            <h2 class="card-title">
+              <Text id="integration.zigbee2mqtt.setup.containersStatus" />
+            </h2>
+          </div>
+          <div class="row justify-content-center d-sm-none">
+            <div class="col-auto">
+              <table class="table table-responsive table-borderless table-sm">
+                <thead class="text-center">
+                  <th>
+                    <Text id="systemSettings.containers" />
+                  </th>
+                  <th>{`Status`}</th>
+                </thead>
+                <tbody class="text-center">
+                  <tr>
+                    <td>{`Gladys`}</td>
```

Comment:
Please prefer i10n files (may exists in a common translation).

Reply:
Fixed